lib/logger.js

```diff
@@ -0,0 +1,3 @@
+var logger = require('@google/cloud-diagnostics-common').logger;
+var logLevel = process.env.GCLOUD_DEBUG_LOGLEVEL;
```

Comment:
I think we will want our own env variable for this rather than tying ourselves to the debug log level.